src/components/MessageList.vue

```diff
@@ -71,6 +71,7 @@ export default {
     },
     preview_format (message) {
       let text = JSON.stringify(message.content.content)
+      if (text === undefined || text === null) { text = "No content" }
       if (text.length > 80) { return text.substring(0, 80) + '...' } else { return text }
```

Comment:
```suggestion
    preview_format (message) {
      const content = message.content.content
      if (content === undefined || content === null) { return "No content" }
      const text = JSON.stringify(content)
      if (text.length > 80) { return text.substring(0, 80) + '...' } else { return text }
```
I think that the check should be before the `JSON.stringify()` method call.